File: DKS-SPA/src/app/core/_models/dev-bom-file-detail-dto.ts
import { ModelInterface } from "./interface/model-interface";

export class DevBomFileDetailDto implements ModelInterface{
     factoryId: string;
     devTeamId: string;
     teamName: string;
     season: string;
     modelNo: string;

     modelName: string;
     article: string;
     stage: string;
     actDate: string;
     fileName: string;

     ver: number;
     sort: number;
     remark: string;
     apply: string;
     //id: number;

     remarkButton: string;
     downloadButton: string;
     applyButton: string;
     uploadButton: string;

     cwaDeadLine: string;
     pdmApply: string;
     ecrNo: string;
     articleList: string;
     dropDate: string;
}

File: DKS-API/DTOs/DevBomFileDetailDto.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DKS.API.Models.DKS
{
    public class DevBomFileDetailDto
    {

    public string FactoryId { get; set; }
    public string DevTeamId { get; set; }
    public string TeamName { get; set; }
    public string Season { get; set; }
    public string ModelNo { get; set; }

    public string ModelName { get; set; }
    public string Article { get; set; }
    public string Stage { get; set; }
    public string ActDate { get; set; }
    public string FileName { get; set; }

    public short Ver { get; set; }
    public short Sort { get; set; }
    public string Remark { get; set; }
    public string Apply { get; set; }
    //public long Id { get; set; }
    public string RemarkButton { get; set; }
    public string DownloadButton { get; set; }
    public string ApplyButton { get; set; }
    public string UploadButton { get; set; }

    public string CwaDeadLine { get; set; }
    public string PdmApply { get; set; }
    public string EcrNo { get; set; }

    }
}